src/components/MapView/Direction.web.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
Please add a comment to explain why there is a difference implementation for `web`

Reply:
Done